packages/data/addon/services/bard-metadata.js

```diff
@@ -38,6 +38,11 @@ export default Service.extend({
    */
   metadataLoaded: false,
 
+  /**
+   * @property {Array} loadedDataSources - list of data sources in which meta data has already been loaded
+   */
+  loadedDataSources: null,
```

Comment:
Is this comment on the right line?

Reply:
It isn't :) This comment was for line 39. This line (44) seems like a replacement for 39

Reply:
we can get rid of it eventually right now that property is used widely as part of the public API. Best we can do is mark it as deprecated.

Reply:
@cythrawll Ah, where do we use it? I couldn't find any references outside this service.

If it is used, can we computed it off `loadedDataSources`? Right now it looks like a single data source would change this to true instead of indicating all metadata is loaded.

Reply:
I could have sworn we used in the apps itself, seems like i'm wrong. Yoink it goes.